src/Groups/GAPGroups.jl

```diff
@@ -1110,7 +1159,8 @@ julia> (length(h), order(h[1]))
 (1, 6)
 
 julia> g = GL(3, 2)
-GL(3,2)
+Matrix group of degree 3
+  over finite field of degree 1 over GF(2)
```

Comment:
I guess the custom printing here should be preserved

Reply:
When should the name be set, only in the construction or also later on when the structure in question has been found out? (The same would hold for symmetric groups.)